From 50db943c234cd650e6930b99f3b2cbf675447aa2 Mon Sep 17 00:00:00 2001
From: Chris Liddell <chris.liddell@artifex.com>
Date: Wed, 17 Apr 2013 09:24:22 +0100
Subject: [PATCH] Raise the default VMThreshold from 1Mb to 8Mb

This seems like a fair change based on modern hardware and, in particular
modern PDF files. In some cases this results in a significant speed
improvement. It may increase overall memory usage, but testing suggests only
by an insignificant amount in the vast majority of cases.

No cluster differences.
---
 psi/zvmem2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/psi/zvmem2.c b/psi/zvmem2.c
index 53fe44b..13d9b6b 100644
--- a/psi/zvmem2.c
+++ b/psi/zvmem2.c
@@ -25,7 +25,7 @@
 
 /* Garbage collector control parameters. */
 #define DEFAULT_VM_THRESHOLD_SMALL 100000
-#define DEFAULT_VM_THRESHOLD_LARGE 1000000
+#define DEFAULT_VM_THRESHOLD_LARGE 8000000
 #define MIN_VM_THRESHOLD 1
 #define MAX_VM_THRESHOLD max_long
 
-- 
2.9.4

